src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/InitMeansTestOutcomeCalculator.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
+
+
+@Slf4j
+public class InitMeansTestOutcomeCalculator {
+
+    public static MeansTestOutcome getInitMeansTestOutcome(InitAssessmentResult initAssessmentResult, Boolean fullAssessmentAvailable){
```

Comment:
as noted by the tests, fullAssessmentAvailable can't really be null so should probably be a boolean